shopify-cli.gemspec

```diff
@@ -4,9 +4,8 @@ Gem::Specification.new do |spec|
   spec.name = "shopify-cli"
   spec.version = ShopifyCli::VERSION
   spec.authors = ["Shopify"]
-  # need to change the email that ends with `shopify.com`
-  spec.email = ["developers@jadedpixel.com"]
-  spec.license = 'Nonstandard'
+  spec.email = ["dev-tools-education@shopify.com"]
```

Comment:
Who does this go to? Just wonder if it may attract spam.

Reply:
It's a Google group e-mail, so it will got to all of us!  It's the e-mail Tim gave me to use
@tanema, Andy does have a point about potentially attracting spam, do we still want to use it?

Reply:
@tanema says he's good with it, similar to the themekit@shopify.com email that's used (gets some spam occasionally but Google's filters do the trick!)